packages/components/Callout/macos/CalloutView.swift

```diff
@@ -1,19 +1,71 @@
 import AppKit
 import Foundation
-#if USE_REACT_AS_MODULE
 import React
-#endif // USE_REACT_AS_MODULE
+
+/// Return the text length of a provided RCTShadowView
+func getLengthOfTextShadowNode(shadowView: RCTShadowView) -> Int {
```

Comment:
nit: A number of these function names are a bit verbose. Swift tends to like to "interleave" a function's name with its arguments when it can, for example, [`convert(_ rect: NSRect, to view: NSView?)`](https://developer.apple.com/documentation/appkit/nsview/1483217-convert).

This is more about style than functionality; this should be totally functional as is.